canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.jsx

```diff
@@ -178,17 +202,20 @@ class RadiosetControl extends React.Component {
 				row: i
 			};
 			buttons.push(
-				<div key={i} className="properties-radioset-panel">
-					<RadioButton
-						key={i}
-						id={ControlUtils.getControlId(id, this.uuid)}
-						disabled={this.props.state === STATES.DISABLED || itemDisabled}
-						labelText={valueSet.valueLabels[i]}
-						value={val}
-						onChange={this.handleChange}
-						checked={checked}
-					/>
-					{optionalPanel}
+				<div className="radioset-tooltip-wrapper" key={ControlUtils.getControlId(id, this.uuid)}>
```

Comment:
Why do we need to wrap this in another div?  This `properties-radioset-panel` should be enough.